### PR TITLE
Add infrastructure team

### PR DESCRIPTION
**Pull request by simpsora**
Adds config for the Infrastructure team.

Is there any way to test this config now?

/cc @johnsyweb

### CODE REVIEW

**Comment by johnsyweb** [comment]
You can test like this: ```sh
[Tue 16/06/07 09:25 AEST][s005][x86_64/darwin15.0.0/15.5.0][5.1.1]
<paj@clacks:~/src/seal>
zsh 10040  (git)-[add-infra]-% ./bin/seal.rb infrastructure
```

File: config/envato.yml [hunk 1]
```diff
@@ -78,3 +78,15 @@ customer-group-leads:
   channel: '#customer-eng-leads'
   repos:
     - customer-group-leads
+
+infrastructure:
+  channel: '#infrastructure'
+  github_team_names: ['Infrastructure']
```
Comment (johnsyweb) [inline config/envato.yml]: Looks like the Seal is struggling to find this team name.

Comment (simpsora) [inline config/envato.yml]: Hmm, perhaps it's a secret team.  I'll remove it from the config.

Comment (johnsyweb) [inline config/envato.yml]: I forgot to export my GitHub API key.

**Comment by simpsora** [comment]
Test worked for me.  Look okay to you @johnsyweb?

**Comment by johnsyweb** [comment]
:shipit:

**Pull request merged**